build_library/disk_util

```diff
@@ -739,12 +755,23 @@ def Verity(options):
       Tune2fsReadWrite(options, part, disable_rw=True)
 
     with PartitionLoop(options, part) as loop_dev:
-      Sudo(['veritysetup', 'format', '--hash=sha256',
-	    '--data-block-size', part['fs_block_size'],
-	    '--hash-block-size', part['fs_block_size'],
-	    '--data-blocks', part['fs_blocks'],
-	    '--hash-offset', part['fs_bytes'],
-	    loop_dev, loop_dev])
+      verityout = SudoPipe(['veritysetup', 'format', '--hash=sha256',
+                  '--data-block-size', part['fs_block_size'],
+                  '--hash-block-size', part['fs_block_size'],
+                  '--data-blocks', part['fs_blocks'],
+                  '--hash-offset', part['fs_bytes'],
+                  loop_dev, loop_dev])
+      root_line = verityout.strip().split("\n")[-1]
+      root_hash = root_line.split("\t")[-1]
```

Comment:
Although this is pretty simple I'd like to do a little more verification of the output. perhaps:

```python
print verityout.strip()
m = re.search("Root hash:\s+([a-f0-9]{64})$", verityout, re.IGNORECASE|re.MULTILINE)
if not m:
    raise Exception("Failed to parse verity output!")
if options.root_hash:
    with open(options.root_hash, "w") as hash_file:
        hash_file.write(m.group(1))
        hash_file.write("\n")
```